src/main/resources/edu/cornell/kfs/vnd/cu-vnd-resources.properties

```diff
@@ -29,4 +29,6 @@ vendor.employee.comparison.report.detail.table.row=\
 %1$13.13s    %2$-11.11s    %3$-11.11s    %4$-10.10s    %5$-6.6s    %6$-10.10s    %7$-15.15s    %8$-24.24s
 vendor.employee.comparison.report.empty.detail.section.message=\
 There were no vendors representing employees that are currently active or were terminated within the past year.
-vendor.workday.call.error=Unable to call WorkDay service to check vendor tax number against WorkDay.
\ No newline at end of file
+vendor.workday.call.error=Unable to call Workday service to check vendor tax number.
+vendor.workday.active=Is an active employee in Workday.
+vendor.workday.terminated=Is an employee that has been terminated within the last {0} days in Workday
```

Comment:
The first two messages end with a period. Should the third message do so as well?

Reply:
ok